src/components/SearchBox.js

```diff
@@ -267,8 +293,7 @@ class SearchBox extends Component {
 
   onSuggestionSelected = suggestion => {
     this.setValue({ value: suggestion && suggestion.value, isOpen: false });
-    this.searchBase &&
-      this.searchBase.triggerClickAnalytics(suggestion && suggestion._click_id);
+    this.triggerClickAnalytics(suggestion && suggestion.source._id);
```

Comment:
It should be the click position, not the source ID.
Click position is the position of you the item in the list, check how we do it in RS.